lib/LaTeXML/Package/pst-node.sty.ltxml

```diff
@@ -180,39 +180,46 @@ DefMacro('\Rnode PSCoord', sub {
     (T_CS('\@Rnode'), T_OTHER('('), Explode($coord), T_OTHER(')')); });
 
 DefConstructor('\@Rnode PSCoord {} {}', '#3',
-  alias => '\Rnode',
+  alias       => '\Rnode',
   afterDigest => sub { storeNode($_[1]->getArg(2), $_[1]->getArg(1)); });
 
 DefConstructor('\pnode ZeroPSCoord {}', '',
   afterDigest => sub { storeNode($_[1]->getArg(2), $_[1]->getArg(1)); });
 
 DefPSConstructor('\cnode', 'ZeroPSCoord {PSDimension} {}',
-  "<ltx:circle " . $PSLineAttr . "%&PairAttr(#2,x,y) r='&ptValue(#3)' fill='#fill' />",
-  attributes => [qw(fill linecolor linewidth dash)],
-  afterDigest => sub { storeNode($_[1]->getArg(4), $_[1]->getArg(2)); });
+  "<ltx:circle " . $PSLineAttr . "x='#x' y='#y' r='&pxValue(#3)' fill='#fill' />",
+  attributes  => [qw(fill linecolor linewidth dash)],
+  afterDigest => sub { storeNode($_[1]->getArg(4), $_[1]->getArg(2)); },
```

Comment:
Not well-versed in `DefPSConstructor`, but I am counting 3 arguments in the pattern above, while I see a `getArg(4)`. 
I see that in the definition of `DefPSConstructor` an `Arrows` argument is always prepended, indeed yielding 4 args total. Slightly too magical... :> But a comment for another day.